server/controllers/index.js

```diff
@@ -144,6 +145,19 @@ export const seriesTransporter = async(ctx, next) => {
   return next();
 };
 
+export const latestTweets = async(ctx, next) => {
```

Comment:
I wonder if we should leave a note, or potentially break this out into something else (`async-controllers`?) to make sure people know not to use these in the sync world due to the massive round trip... just a thought, happy not to for now.